core/node/libp2p/rcmgr_defaults.go

```diff
@@ -38,14 +38,17 @@ func adjustedDefaultLimits(cfg config.SwarmConfig) rcmgr.DefaultLimitConfig {
 	if cfg.ConnMgr.Type == "basic" {
 		maxconns := cfg.ConnMgr.HighWater
 		if 2*maxconns > defaultLimits.SystemBaseLimit.ConnsInbound {
-			// adjust conns to 2x to allow for two conns per peer (TCP+QUIC)
+			// Conns should be at least 2x larger than the high water to allow for two conns per peer (TCP+QUIC).
+			//
+			// Outbound conns are set very high to allow for the accelerated DHT client to refresh its routing table.
+			// Currently it does not gracefully handle RM throttling, once it does we can lower this.
 			defaultLimits.SystemBaseLimit.ConnsInbound = logScale(2 * maxconns)
-			defaultLimits.SystemBaseLimit.ConnsOutbound = logScale(2 * maxconns)
-			defaultLimits.SystemBaseLimit.Conns = logScale(4 * maxconns)
+			defaultLimits.SystemBaseLimit.ConnsOutbound = logScale(64 * maxconns)
```

Comment:
This doesn't seem sufficient to effectively turn off outbound limits. Local debugging still shows errors due to a cap of ~2k connections, I'm not sure which limit this is coming from though.